src/ol/format/KML.js

```diff
@@ -311,14 +307,23 @@ export function getDefaultStyleArray() {
   return DEFAULT_STYLE_ARRAY;
 }
 
+/**
+ * Function that returns the scale needed to normalize an icon image to 32 pixels.
+ * @param {import("../size.js").Size} size Image size.
+ * @return {number} Scale.
+ */
+function resizeScaleFunction(size) {
```

Comment:
I know there is precedent elsewhere in the library, but I think we should stop naming functions with the word `Function` in them.  This could be `scaleForSize`.